oreClient/src/main/resources/assets/components/Pagination.vue

```diff
@@ -0,0 +1,84 @@
+<template>
```

Comment:
Can we make a prettier pagination for the home page? It just doesn't feel like it fits at all. At least make it a bit bigger.

Reply:
Yeah just copied the design for the version page